src/Uno.UWP/UI/Core/ICoreWindowsEvents.iOS.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Windows.UI.Core
+{
+	public interface ICoreWindowEvents
```

Comment:
Does it need to be public?

Reply:
Yes. The parent [`ICoreWindowEvents`](https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Core/ICoreWindowEvents.cs#L7) interface is public